mkdocs.yml

```diff
@@ -11,6 +11,7 @@ nav:
         - Introduction: validators/introduction.md
         - "Http User Agent": validators/httpuseragent.md
         - "Remote Address": validators/remoteaddr.md
+        - "CSRF Validator": validator/csrf.md
```

Comment:
Please use the folder `validators`.

Reply:
Thanks @froschdesign - should be good to go now